internal/observerware/middleware_test.go

```diff
@@ -59,49 +61,113 @@ func TestUnaryInboundMiddleware(t *testing.T) {
 		wantFields []zapcore.Field
 	}{
 		{
-			desc:    "downstream errors",
+			desc:    "no downstream errors",
 			handler: fakeHandler{},
 			extract: NewNopContextExtractor(),
 			wantFields: []zapcore.Field{
-				zap.String("rpcType", "unary inbound"),
+				zap.String("rpcType", "unary"),
 				zap.Skip(),
-				zap.Object("request", req),
+				zap.Object("request", _req),
 				zap.Duration("latency", 0),
 				zap.Bool("successful", true),
 				zap.Skip(),
 			},
 		},
 		{
+			desc:    "downstream errors",
+			extract: NewNopContextExtractor(),
+			handler: fakeHandler{errFailed},
+			wantErr: true,
+			wantFields: []zapcore.Field{
+				zap.String("rpcType", "unary"),
+				zap.Skip(),
```

Comment:
I’m curious about skip fields. Sourcegraph tells me "Skip constructs a no-op field" but not what they’re for. I gather they’re effectively "AnyField" matchers for testing.